CONTENT/schedule.markdown

```diff
@@ -9,12 +9,12 @@ comment: goal - event emitter + bookmark bar app to subscribe
 
 ---
 [](#english)
-# Sessions of Wizardry
+# Magic sessions
```

Comment:
I changed that one, because later below it says:
* Super Magic Sessions
* Magic Sessions
* Show'N'Tell
* Free Trial Session
So I thought about a headline for those different types of events, and maybe "Magic Sessions" might not cut it.
What do you think?